### DISCUSSION TITLE
Welcome to freeelectronicssamples Discussions!

### DISCUSSION THREAD

**Discussion by jamesy0ung** [jamesy0ung/freeelectronicssamples#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Please keep the discussion on-topic and contribute with things to add.
Thank You!

**Comment by NNNILabs** [comment]
Can we add information about getting samples from CoilCraft and update the ADI section? I have written up the following:

CoilCraft: Inductors, Transformers and other high-quality magnetics. They are very generous with samples. You can contact a sales rep, and they will provide you with samples regardless of if you are working, a student or just an independent hobbyist. Contacting them through a regular E-Mail domain (e.g. @gmail.com) also works.

Analog Devices: Samples only if you have a corporate/university E-Mail ID. Usually require an EORI number for EU shipping (you can fill in the field with N/A and mention you are a student) but this might get you in trouble with customs. You are required to fill in details about what project the samples will be used in. Only _one_ samples account per person is allowed. Limit samples orders to two parts of each type, and five types in total.

**Reply by jamesy0ung** [reply]
I have incorporated your changes, thank you!

**Reply by NNNILabs** [reply]
You're welcome, glad to contribute!
